website/docs/r/ip_space.html.markdown

```diff
@@ -166,9 +198,14 @@ The following arguments are supported:
   ([ip_range](#ipspace-ip-range)), enter a number of floating IP addresses to allocate individually.
   `-1` is unlimited, while `0` means that no IPs can be allocated.
 * `ip_prefix` - (Optional) One or more IP prefixes (blocks) [ip_prefix](#ipspace-ip-prefix)
-
 * `route_advertisement_enabled` - (Optional) Toggle on the route advertisement option to
   enable advertising networks with IP prefixes from this IP space (default `false`)
+* `default_firewall_rule_creation_enabled` - (Optional, *v3.11+*, *VCD 10.5.0+*) Defines whether
+  default firewall rule creation should be enabled
+* `default_no_snat_rule_creation_enabled` - (Optional, *v3.11+*, *VCD 10.5.0+*) Defines whether NO SNAT
+  rule creation should be enabled
+* `default_snat_rule_creation_enabled` - (Optional, *v3.11+*, *VCD 10.5.0+*) Defines whether SNAT rule
+  creation should be enabled
```

Comment:
I'm thinking maybe it's worth adding a sentence on NO SNAT terminology, as it feels an advanced topic. Maybe this:

```suggestion
* `default_no_snat_rule_creation_enabled` - (Optional, *v3.11+*, *VCD 10.5.0+*) Defines whether NO SNAT
  rule creation should be enabled. NO SNAT prevents SNAT address translation.
```

BTW, there are no defaults for NO DNAT/DNAT here, right?

Or we could even make a precedent and give a link to govcd docs (not sure how to keep them in sync in the long run though):
https://github.com/vmware/go-vcloud-director/pull/628/files#diff-91f329c83083b4e72429075d5f926a8b4dc4f47363c52e17c1f2b950a0edeeb1R104-R122
In general, such links to govcd type docs (comments) could be useful in some other places too, as they usually have more description. Just an idea for future consideration. This would also expose govcd more. CC: @dataclouder @adambarreiro @adezxc

Reply:
> BTW, there are no defaults for NO DNAT/DNAT here, right?

Nope, this is all we've got in the API https://developer.vmware.com/apis/vmware-cloud-director/latest/data-structures/IpSpaceDefaultGatewayServiceConfig/


> In general, such links to govcd type docs (comments) could be useful in some other places too, as they usually have more description.

It is possibly cleaner to link to VCD API public docs (I usually take the information from there for govcd structs).
It does mask govcd, but possibly persists better